docs/adr/adr-010-incentivized-testnet-monitoring.md

```diff
@@ -167,14 +168,44 @@ The Prometheus docs state the following with regard to [Denial of Service](https
 
-### Why doesn't the Celestia team host OTEL Collectors for node operators?
+### Which actor should run OTEL Collector(s) during the incentivized testnet?
 
-1. Node operators will lose the ability to monitor their own celestia-node. Since opentelemetry-go only supports configuring one exporter ( [open-telemetry/opentelemetry-go#3055](https://github.com/open-telemetry/opentelemetry-go/issues/3055)), if node operators were obligated to export metrics to a Celestia team managed OTEL Collector endpoint, they wouldn't be able to export to their own OTEL Collector (and by proxy any telemetry platforms they wish to use). This violates the desideratum:
+#### Scenario A: Node operators
+
+Pros
+
+- This deployment architecture is more representative of mainnet where node operators will run their own telemetry stack to monitor their node. Exposing node operators to OTEL Collector during incentivized testnet allows them to practice this deployment architecture prior to mainnet.
+- Node operators will have an "incentive" to maintain high uptime for their OTEL Collector.
+
+Cons
+
+- Additional operational burden for incentivized testnet participants. We can mitigate this concern by providing easy install steps and scripts.
+
+#### Scenario B: Celestia team
+
+Pros
+
+- It will be easier for nodes to participate if they only have to deploy one piece of software (celestia-node) and not two (celestia-node and OTEL Collector).
+
+Cons
+
+- Node operators will lose the ability to monitor their own celestia-node. Since opentelemetry-go supports configuring only one exporter ( [open-telemetry/opentelemetry-go#3055](https://github.com/open-telemetry/opentelemetry-go/issues/3055)), if node operators were obligated to export metrics to a Celestia team managed OTEL Collector endpoint, they wouldn't be able to export to their own OTEL Collector (and by proxy any telemetry platforms they wish to use). This violates the desideratum:
 
     > We would like to make it possible for node operators to monitor their own nodes with existing telemetry tools (e.g. Grafana and Uptrace)
 
-1. Node operators will have an "incentive" to maintain high uptime for their OTEL Collector. If the Celestia team took on this responsibility and failed to provide a highly available solution, then node operators would be penalized for downtime of a component they have no control over.
-1. We expect 1500+ node operators during the incentivized testnet and there is minimal documentation on the scale of workload an individual OTEL Collector can handle. We'd have to design and operate a highly available OTEL Collector fleet to maintain high uptime for node operators. At this time no cloud managed offerings for OTEL Collector exist.
+- If the Celestia team took on this responsibility and failed to provide a highly available solution, then node operators would be penalized for downtime of a component they have no control over.
```

Comment:
Well, if there is downtime, we would know this and would definitely not penalize because of our fault. I would just say it's an additional burden for us to maintain infrastructure here as a con

Reply:
Done in [7ffa115](https://github.com/celestiaorg/celestia-node/pull/922/commits/7ffa115337aa21e62177483e68c64763041cf615)